src/components/search/search.style.js

```diff
@@ -41,125 +42,151 @@ const StyledSearch = styled.div`
+        css`
+          border-color: transparent;
+          color: ${theme.icon.defaultHover};
+        `
+      }
+      ${
+        !isFocused &&
+        searchHasValue &&
+        !showSearchButton &&
+        css`
+          border-bottom: 2px solid ${variantColor};
+          :hover {
+            border-bottom-color: ${theme.search.active};
+            cursor: pointer;
           }
-        `}
+        `
+      }
 
-        ${darkVariant &&
-        css`
-          ${!isFocused &&
-          searchHasValue &&
-          !showSearchButton &&
+      ${StyledInput} {
+        ${
+          darkVariant &&
+          !isFocused &&
           css`
-            color: ${theme.search.darkVariantText};
-          `}
-          ${!isFocused &&
-          !searchHasValue &&
+            ::-moz-placeholder {
+              color: ${theme.search.darkVariantPlaceholder};
+              opacity: 1;
+            }
+            ::placeholder {
+              color: ${theme.search.darkVariantPlaceholder};
+            }
+          `
+        }
+
+        ${
+          darkVariant &&
           css`
-            color: ${theme.search.darkVariantPlaceholder};
-          `}
-        `}
+            ${!isFocused &&
+            searchHasValue &&
+            !showSearchButton &&
+            css`
+              color: ${theme.search.darkVariantText};
+            `}
+            ${!isFocused &&
+            !searchHasValue &&
+            css`
+              color: ${theme.search.darkVariantPlaceholder};
+            `}
+          `
+        }
       }
 
       ${StyledInputPresentation} {
-        background-color: ${searchHasValue || isFocused
-          ? `${theme.colors.white}`
-          : "transparent"};
+        background-color: ${
+          searchHasValue || isFocused ? `${theme.colors.white}` : "transparent"
+        };
         flex: 1;
         font-size: 14px;
         font-weight: 700;
         padding-bottom: 2px;
         padding-top: 1px;
         cursor: pointer;
-        ${!isFocused &&
-        !searchHasValue &&
-        css`
-          border: 1px solid transparent;
-          color: ${theme.icon.default};
-        `}
-        ${!isFocused &&
-        searchHasValue &&
-        !showSearchButton &&
-        css`
-          border: 1px solid transparent;
-          background-color: ${darkVariant
-            ? "transparent"
-            : `${theme.colors.white}`};
-        `}
+        ${
+          !isFocused &&
+          !searchHasValue &&
+          css`
+            border: 1px solid transparent;
+            color: ${theme.icon.default};
+          `
+        }
+        ${
+          !isFocused &&
+          searchHasValue &&
+          !showSearchButton &&
+          css`
+            border: 1px solid transparent;
+            background-color: ${darkVariant
+              ? "transparent"
+              : `${theme.colors.white}`};
+          `
+        }
       }
 
       ${StyledFormField} {
+        margin-bottom: 0px
```

Comment:
Maybe we should consider having a test to cover this bug? I don't think it's captured in Chromatic and there isn't a unit test to assert the styles anywhere. Currently if someone were to make a change to Search or Form and reintroduce this bug, we wouldn't really know about it. What do you think @grabkowski ?